test/integration/DCAKeep3rJob/keep3r-job.spec.ts

```diff
@@ -25,7 +25,7 @@ contract('DCAKeep3rJob', () => {
   let WETH: IERC20, K3PR: IERC20;
 
   let DCAKeep3rJob: DCAKeep3rJob;
-  let DCAHubSwapper: DCAHubSwapper;
+  let DCAHubSwapper: ThirdPartyDCAHubSwapper;
```

Comment:
Any chance of changing the name to the 3PartySwapper so the test is more explicity?

```suggestion
  let DCAHubSwapper: ThirdPartyDCAHubSwapper;
```

Reply:
What do you want to rename exactly?

Reply:
Oops.

```suggestion
  let thirdPartySwapper: ThirdPartyDCAHubSwapper;
```

Reply:
Done!